<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPImage</class>
 <widget class="QMainWindow" name="FPImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FPImage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="ERes"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="rInfinito">
           <property name="text">
            <string>norma infinito</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rManhattan">
           <property name="text">
            <string>norma Manhattan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rNorma2">
           <property name="text">
            <string>norma 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QRadioButton" name="rCuadrado">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>0</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Cuadrado</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rCirculo">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>0</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Circulo</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="bEtiquetar">
        <property name="text">
         <string>Etiquetar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bErosion">
        <property name="text">
         <string>Erosion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bDilatacion">
        <property name="text">
         <string>Dilatacion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bFiltrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bPiel">
        <property name="text">
         <string>Reconocedor facial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bBW">
        <property name="text">
         <string>B/W</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEcualizar">
        <property name="text">
         <string>Ecualizador</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bStretch">
        <property name="text">
         <string>Stretch Lineal</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="BLoad">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BDoIt">
          <property name="text">
           <string>Do it</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>586</width>
         <height>626</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="Ecran">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="EcranHistoR">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>104</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>104</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="EcranHistoG">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>104</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>104</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="EcranHistoB">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>104</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>104</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Rpiel">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>110</number>
        </property>
        <property name="sliderPosition">
         <number>110</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="Rdesv">
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="sliderPosition">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Gpiel">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>113</number>
        </property>
        <property name="sliderPosition">
         <number>113</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Gdesv">
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="value">
         <number>89</number>
        </property>
        <property name="sliderPosition">
         <number>89</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Bpiel">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>14</number>
        </property>
        <property name="sliderPosition">
         <number>14</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Bdesv">
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
        <property name="sliderPosition">
         <number>8</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sBordes">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sContraste">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>890</number>
        </property>
        <property name="sliderPosition">
         <number>450</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sBrillo">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sDim">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
